docs/api/remix.md

```diff
@@ -2055,6 +2055,8 @@ const { getSession, commitSession, destroySession } =
 export { getSession, commitSession, destroySession };
 ```
 
+Please note that using the `createCookieSessionStorage` code above as it stands is NOT recommended. Not only are `expires` and `maxAge` not intended for use together, but using this code would set a specific, one-off time for all future cookies, which would eventually elapse preventing any futher cookies being set!
```

Comment:
I've addressed it by commenting out expires and adding a note. It seems useful to leave the sample in rather than suggest `expires` isn't even an option.